lib/phx_tools_web/live/phx_tools_live/index.ex

```diff
@@ -35,24 +35,15 @@ defmodule PhxToolsWeb.PhxToolsLive.Index do
     {:noreply,
      socket
      |> assign(seo_attributes: %{url: Endpoint.url() <> "/#{action}"})
-     |> assign(:source_code_url, source_code_url_from_os_or_live_action(action))}
+     |> assign(:source_code_url, source_code_url())}
   end
 
-  defp installation_command(live_action) do
-    "$SHELL -c \"$(curl -fsSL #{get_script_url(live_action)})\""
+  defp installation_command do
+    "$SHELL -c \"$(curl -fsSL #{Endpoint.url()})\""
   end
 
-  defp source_code_url_from_os_or_live_action(os_or_live_action) do
-    ~S(https://github.com/optimumBA/phx.tools/blob/main/priv/static/) <>
-      get_script_filename(os_or_live_action)
-  end
-
-  defp get_script_url(live_action_or_os) do
-    "#{Endpoint.url()}/#{get_script_filename(live_action_or_os)}"
-  end
-
-  defp get_script_filename(live_action_or_os) do
-    "#{get_operating_system(to_string(live_action_or_os))}.sh"
+  defp source_code_url do
+    ~S(https://github.com/optimumBA/phx.tools/blob/main/priv/script.sh)
```

Comment:
Use simple quotes unless necessary.
```suggestion
  "https://github.com/optimumBA/phx.tools/blob/main/priv/script.sh"
```